src/screens/discover/DiscoverDetailScreen.tsx

```diff
@@ -104,26 +105,27 @@ const DiscoverDetailScreen: React.FC = () => {
         );
     }, [data, hash, viewState, profile]);
 
-    // TODO: Create loader #839r
-    if (loading) return <p>Loading...</p>;
-
     return (
         <div
             className='flex flex-col items-center w-full m-6'
             data-testid={`discover-details-${path}-screen`}
         >
             <Typography variant='h4'>{title}</Typography>
-            <DetailScreen
-                viewStateKey={viewStateKey}
-                viewState={viewState}
-                setViewState={setViewState}
-                data={data}
-                setData={setData}
-                setHash={setHash}
-                cards={cards}
-                disableAlphabeticOrderFilter={true}
-                disableResultTypeFilter={true}
-            />
+            {loading ? (
+                <CardLoader query={title} viewState={viewState} />
```

Comment:
This looks really good! I think the only "problem" I see is that the filter bar is not in the loader. This not a deal breaker, but it does cause the UI to jump a bit between the loader and the content. If it is not too much to add, could you add the filter to the loader screen? `SearchResultLoader` has a good example of this. You may need to create a wrapper, something like `DiscoverDetailLoader`.

Reply:
![Screenshot 2024-11-30 at 17-34-55 Streamability](https://github.com/user-attachments/assets/8cbebc1c-2cf1-404b-9e01-3fed4484a5b2)
![Screenshot 2024-11-30 at 17-34-19 Streamability](https://github.com/user-attachments/assets/717d1be2-87c6-4c5c-96e3-6390d2c0eda3)

Should be good to go here as well. Provided updated images for the layout. Hope you had a great thanksgiving!